manifests/spf.pp

```diff
@@ -7,4 +6,16 @@
   package { 'postfix-policyd-spf-perl':
     ensure => present,
   }
+
+  file { '/etc/postfix/master.cf':
+    ensure  => file,
+    owner   => root,
+    group   => root,
+    mode    => '0644',
+    source  => 'puppet:///modules/postfix/master.cf',
```

Comment:
This won't work, cause `master.cf` is already being managed by this module, here: https://github.com/lutak-srce/postfix/blob/master/manifests/init.pp#L220

Hence you would get a resource conflict.